ios/Plugin/AudioAsset.swift

```diff
@@ -171,4 +171,14 @@ public class AudioAsset: NSObject, AVAudioPlayerDelegate {
     func playerDecodeError(player: AVAudioPlayer!, error: NSError!) {
         
     }
+
+    func isPlaying() -> Bool {
+        if channels.count != 1 {
+            return 0
```

Comment:
here it should be `return false` it will through the error as expect return type should be of Boolean

Reply:
Thanks, appreciate it